DigitalLearningSolutions.Data/Services/UserService.cs

```diff
@@ -1,13 +1,16 @@
 ﻿namespace DigitalLearningSolutions.Data.Services
 {
     using System.Collections.Generic;
+    using System.Linq;
     using DigitalLearningSolutions.Data.Models.User;
 
     public interface IUserService
     {
         public (AdminUser?, List<DelegateUser>) GetUsersByUsername(string username);
         public (List<AdminUser>, List<DelegateUser>) GetUsersByEmailAddress(string emailAddress);
         public (AdminUser?, DelegateUser?) GetUsersById(string? adminId, string? delegateId);
+
+        public List<CentreUserDetails> GetAvailableCentres(AdminUser adminUser, List<DelegateUser> delegateUsers);
```

Comment:
I'm thinking that the name GetUserCentres might work better here, as "Available" feels more ambiguous, but this could be subjective.